src/main/java/com/fvv/bookstore/bean/Book.java

```diff
@@ -179,15 +211,33 @@ public void setGenre(String genre) {
 	 */
 	public String getGenre() {
 		return this.genre;
+	}	
+	
+	/**
+	 * Sets the date.
+	 * 
+	 * @param date the date to set
+	 */
+	public void setDate(Date date) {
+		this.date = date;
+	}
+
+	/**
+	 * Gets the last date.
+	 * 
+	 * @return the last date of Date type.
+	 */
+	public Date getDate() {
+		return this.date;
 	}
 
 	/**
 	 * @see java.lang.Object#toString()
 	 */
 	public String toString() {
-		return "Book Title: " + getTitle() + ", Publication Year: " + getPublicationYear() + 
-				", Edition Number: " + getEditionNumber() + ", Author: " + getAuthor() + 
-				", Price: " + getPrice() + ", ISBN: " + getIsbn() + ", Publisher: " + 
-				getPublisher() + ", Genre: " + getGenre();
+		return "Book ID: " + getId() + ", Title: " + getTitle() + ", Publication Year: " + 
+				getPublicationYear() + ", Edition Number: " + getEditionNumber() + ", Author: " + 
+				getAuthor() + ", Price: " + getPrice() + ", ISBN: " + getIsbn() + ", Publisher: " 
+				+ getPublisher() + ", Genre: " + getGenre() + ", Date: " + getDate();
```

Comment:
do you really need `getDate()` in toString? if yes, you need to convert to a friendly format since now this returns a long number. If not, simply remove it from here (consider review all beans)

Reply:
I dont need, I will remove!